homeassistant/components/yale_smart_alarm/config_flow.py

```diff
@@ -27,7 +27,6 @@
     {
         vol.Required(CONF_USERNAME): cv.string,
         vol.Required(CONF_PASSWORD): cv.string,
-        vol.Required(CONF_NAME, default=DEFAULT_NAME): cv.string,
```

Comment:
If the user input is no longer going to include CONF_NAME, then probably [this line](https://github.com/home-assistant/core/blob/7d949a766511240df610c7a6c88ad32146e2461b/homeassistant/components/yale_smart_alarm/config_flow.py#L115) needs to change also.

Reply:
That line is removed hence it's red and marked for removal

Reply:
Yes, it's the linked line that needs to be looked at (it's not on the diff so I couldn't add a comment to the line directly).

Reply:
Ok. I got what you meant finally.
The other line I kept for yaml import but you're right. Should change that one.